witms/pipelines.py

```diff
@@ -6,8 +6,22 @@
 
 # useful for handling different item types with a single interface
 # from itemadapter import ItemAdapter
+from elasticsearch import Elasticsearch, helpers
 
+BUFFER_SIZE = 1000
+
+
+class ElasticSearchPipeline:
+    def __init__(self):
+        self.items_buffer = []
+
+    def open_spider(self, spider):
+        self.client = Elasticsearch(hosts=["localhost"])
 
-class WitmsPipeline:
     def process_item(self, item, spider):
+        action = {"_index": "news-index", "_source": dict(item)}
```

Comment:
Please review this hardcoded index name.

Reply:
Solved on commit 0b757fe